src/gui/folder.cpp

```diff
@@ -688,6 +688,7 @@ void Folder::slotWatchedPathsChanged(const QSet<QString> &paths, ChangeReason re
     Q_ASSERT(isReady());
     for (const auto &path : paths) {
         if (!FileSystem::isChildPathOf(path, this->path())) {
+            Q_ASSERT(false);
```

Comment:
If this is a critical issue, why is the log message just on debug level?

Reply:
Because it has no impact.
It should just not happen, looking at the code I can probably remove the check...